### ISSUE TITLE
Flutter Plugin on Android Studio crashed

### ISSUE THREAD

**Issue by eegoulart** [flutter/flutter-intellij#3720]
Thanks for the feedback! If your issue is related to the Flutter framework itself,
please open an issue at
[github.com/flutter/flutter](https://github.com/flutter/flutter/issues/new).

## Steps to Reproduce

_Please tell us what you were doing and what went wrong_

## Version info

Please paste the output of running `flutter doctor -v` here (available from the command
line or from `Tools > Flutter > Flutter Doctor`). It will provide the version of the
Flutter framework as well as of the IntelliJ plugin.

--- July 31st, 2019 ---
## What happened

(please describe what you were doing when this exception occurred)

## Version information

Android Studio `3.4.2` • Flutter plugin `io.flutter 37.1.1` • Dart plugin `183.6270`

Error getting Flutter sdk information.

## Exception

null

```
java.lang.Throwable: Flutter device daemon #1: process exited during startup. Exit code: 1, stderr:

Oops; flutter has exited unexpectedly.
Crash report written to /Users/user/flutter/flutter_04.log;
please let us know at https://github.com/flutter/flutter/issues.

	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:137)
	at io.flutter.run.daemon.DeviceDaemon$Command.start(DeviceDaemon.java:240)
	at io.flutter.run.daemon.DeviceService.chooseNextDaemon(DeviceService.java:225)
	at io.flutter.utils.Refreshable.runInBackground(Refreshable.java:221)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by jacob314** [comment]
This appears to a crash starting the flutter application. Do you have any more information on how the flutter app crashed?

**Comment by eegoulart** [comment]
I restarted everything and it worked.
Tks.

**Comment by pq** [comment]
Closing since resolved but possibly related to #3797.
